### PR TITLE
Bootstrap project

### PR DESCRIPTION
**Pull request by abrormukimov**
## Newsweek homepage clone

In this project, I used:
- Bootstrap classes;
- flexbox and grid to make responsive;
- media queries to be able to see the page on different devices

### CODE REVIEW

**Comment by mikenath223** [comment]
### Pull Request currently under review by Michgolden Ukeje (TSE)
Kindly exercise patience 🐱

File: index.html [hunk 1]
```diff
@@ -0,0 +1,1031 @@
...
+        </li>
+        <li class="nav-item flex-fill">
+          <a class="#" href="#">Vantage</a>
+        </li>
+        <li class="nav-item flex-fill">
+          <a class="#" href="#">Weather</a>
+        </li>
+      </ul>
+      <form class="my-0 d-flex mr-5">
+        <input class="form-control m-sm-0 m-md-0 m-lg-0 border-0 p-0" style="height: 1.2em; width: 6em;" type="search" placeholder="Search" aria-label="Search">
```
Comment (mikenath223) [inline index.html]: Kindly remove all inline styling similar to this. Consider moving them to your stylesheet.

Comment (abrormukimov) [reply index.html]: Thank you for your review.
It is my pleasure to remind you that I used inline styles for the fact that this is a project with Bootstrap classes and sometimes it is impossible to override the codes in bootstrap if you don't use inline styles. 
!important helped me to override the codes. I didn't get the expected result when I tried to use without them.
For all my previous projects, I didn't use any inline styles and !important. This is only for the reason I am working with Bootstrap classes. Hopefully, I will not use any of them when I master my knowledge with Bootstrap.

**Comment by uryelah** [comment]
Hello **Abror Mukimov**, I'm Sarah, your reviewer for this PR 👩‍🚀  

First of all, good job with the project so far. It looks quite good and responds well with minor details needing fixing only.

Now on the in-line-styling issues, I know that working with bootstrap styles can be tricky, and while I think it's ok to have a couple of `!important`s in the style in this particular case, the inline ones really can't pass like mentioned by the last TSE.
This can be a bit frustrating but please review this [article on MDN on CSS specificity ](https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity) and see if that helps you with the overwriting.

The bootstrap project is one of the most challenging ones in the HTML curriculum, but I'm sure that you can do it! 💪 

---------    

### Required changes 🛠 
---------
 - [ ] As mentioned by the last TSE, do not use inline styles. Check this [StackOverflow thread](https://stackoverflow.com/questions/50138101/external-stylesheet-not-overriding-bootstrap) on overriding Bootstrap styles for ideas on how you can do it.

- [ ] The logo background color changed but it's still different from the background color of the nav itself. Make sure the navbar only has one background color.
Screenshot_2020-03-25 Newsweek
Screenshot_2020-03-25 Newsweek(1)

- [ ] Make sure the collapsed navbar menu icon is in the correct light red color instead of grey. You can use a color picker in your browser of choice as the previous TSE suggested.
-----------

- [ ] The navbar should take 100% of the screen width on small devices too. Compare how your navbar looks and the original one below:
**Your navbar**
Screenshot_2020-03-25 Newsweek(5)
**Original navbar**
Screenshot_2020-03-25 Newsweek(4)


-------
- [ ] The Newsletter image is still deforming on small screens. Make sure it resizes but keeps its proportions correctly:
**Your version:**
Screenshot_2020-03-25 Newsweek(3)
**Original one:**
Screenshot_2020-03-25 Newsweek(2)


--------

**Kindly fix the REQUIRED issues and send another code review request.**

Cheers and Happy coding!👏👏👏

Code review by **Sarah Uryelah Silva Chamorro**

(Feel free to contact me on slack by the handle of  **sarah** if you have any additional question)

**Comment by abrormukimov** [comment]
Changed everything required

File: index.html [hunk 2]
```diff
@@ -0,0 +1,1032 @@
...
+                      <a href="#" class="m-0 p-0 text-dark txt dd">The Ethics of the Coronavirus Crisis Are Muddier Than You Think</a>
+                      <figure class="m-0 p-0 text-secondary">BY CHARLIE KIRK</figure>
+                    </div>
+                  </div>
+                </div>
+              </div>
+              <div class="mt-4  position-relative">
+                <div class="text-secondary">THE DEBATE</div>
+                <div class="mt-2">
+                  <div class="media pb-4" style="border-top: 4px solid rgb(216, 56, 168); border-bottom: 1px solid black;">
```
Comment (uryelah) [inline index.html]: - [ ] Remember to remove **all inline styles** (1)

File: index.html [hunk 3]
```diff
@@ -0,0 +1,1032 @@
...
+                    <a href="#"><img class="align-self-center mr-2" src="images/farage-johnson-corona.webp" style="width: 6em; height: 6em;" alt="Generic placeholder image"></a>
```
Comment (uryelah) [inline index.html]: - [ ] Remember to remove **all inline styles** (2)

File: index.html [hunk 4]
```diff
@@ -0,0 +1,1032 @@
...
+          </div>
+        </div>
+      </div>
+      
+      <div class="in-the-mag text-dark text-center mt-3">
+        <div class="n-logo px-3 py-1 mr-2 bck-r-d-in"></div>
+        FEATURED SLIDESHOWS
+      </div>
+      <div class="container mt-2">
+        <div class="row  border border-right-0 border-left-0 border-bottom-0" style="max-width: 98%; margin: auto;">
```
Comment (uryelah) [inline index.html]: - [ ] Remember to remove **all inline styles** (3)

**Comment by uryelah** [comment]
--------

**Kindly fix the REQUIRED issues and send another code review request.**

Cheers and Happy coding!👏👏👏

Code review by **Sarah Uryelah Silva Chamorro**

(Feel free to contact me on slack by the handle of  **sarah** if you have any additional question)

**Pull request merged**